tsconfig.json

```diff
@@ -15,7 +15,7 @@
     "resolveJsonModule": true,
     "isolatedModules": true,
     "noEmit": true,
-    "jsx": "react-jsx"
+    "jsx": "preserve",
   },
```

Comment:
why preserve?

Reply:
what is it for? I ask out of curiosity

Reply:
I put preserve in because there were errors with react-jsx. I read somewhere that this may help, the bugs disappeared, so I left it, but I don't know if it's right 🙈